src/Roslyn.Diagnostics.Analyzers/Core/Extensions/IMethodSymbolExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Concurrent;
+using Microsoft.CodeAnalysis;
+
+namespace Roslyn.Diagnostics.Analyzers.Extensions
+{
+    internal static class IMethodSymbolExtensions
```

Comment:
Same for this - consider moving to existing extension method file in shared analyzer utilities project.